api/interactives.go

```diff
@@ -147,9 +147,9 @@ func (api *API) UpdateInteractiveHandler(w http.ResponseWriter, req *http.Reques
 		log.Error(ctx, "Error reading body (unmarshal)", ErrInvalidBody)
 		return
 	}
-	if update.ImportSuccessful == nil || update.Interactive.Metadata == nil {
+	if update.ImportSuccessful == nil && update.Interactive.Metadata == nil {
```

Comment:
not really keen on this tbh - think we should use validators on json model `required` or some rules